vyper/semantics/types/module.py

```diff
@@ -232,14 +232,16 @@ def from_ModuleT(cls, module_t: "ModuleT") -> "InterfaceT":
     @classmethod
     def from_InterfaceDef(cls, node: vy_ast.InterfaceDef) -> "InterfaceT":
         functions = []
-        for node in node.body:
+        for sub_node in node.body:
```

Comment:
The node variable is used as a return value, so it was just returning the last one in the body. Now it's returning the whole node.

Reply:
nice catch!

i think a better name though might be `func_ast`